src/libs/KeyboardAvoidingView/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * This is a KeyboardAvoidingView only enabled for ios && disabled for all other platforms
```

Comment:
Hmm, is this one only for ios? What's the difference between this one and `index.ios.js` then?

Reply:
enabled prop is different, that way it only works on ios